cmd/commands/service.go

```diff
@@ -372,7 +377,7 @@ func ServiceSubscribe(fcClient *core.Client) *cobra.Command {
 
 	command := &cobra.Command{
 		Use:     "subscribe",
-		Short:   "Subscribe a service to an existing input channel",
+		Short:   "Subscribe a service to an existing input or output channel",
```

Comment:
Actually, not an "or", as the input is required AFAIU.

More like "and an optional output channel"

Reply:
Fixing it now.